ui/pages/confirmations/components/fee-details-component/fee-details-component.js

```diff
@@ -86,6 +78,8 @@ export default function FeeDetailsComponent({
     [txData, useNativeCurrencyAsPrimaryCurrency],
   );
 
+  const hasLayer1GasFee = layer1GasFee !== null;
```

Comment:
`hasLayer1GasFee` naming is a bit confusing. My thinking is that if `layer1GasFee` is `0` (not sure if this is even possible, as per the controller it either return an hex string prefixed with 0x or undefined), then it would mean that it does not have a `layer1GasFee`, and in fact we are not rendering the `layer1Fees` in the jsx below when `layer1GasFee` is `falsy`.

Reply:
I'm considering `has` to mean any value, even zero.

Supporting `0x0` was intentional as that would imply the transaction supported layer 1 fees and thus it's meaningful information for the user that it's zero, given other transactions (maybe on the same network) could be greater than zero.